OrbitGl/App.cpp

```diff
@@ -724,13 +735,13 @@ bool OrbitApp::OnLoadCapture(const std::string& file_name) {
 
   CaptureSerializer ar;
   ar.time_graph_ = GCurrentTimeGraph;
-  bool loaded = ar.Load(file_name);
-  if (loaded) {
+  outcome::result<void, std::string> result = ar.Load(file_name);
```

Comment:
You could use `OUTCOME_TRY(ar.Load(file_name))`. It returns the error if one occured or continues execution.